# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/ScreenHelperGTK.h
# Commit: f562693b5b30
# Full Hash: f562693b5b30344af6419b1668bcbcf3a804daba
# Author: stransky <stransky@redhat.com>
# Date: 2025-07-08 16:25:02
# Regressor Bug: 1971161
# File Overlap Count: 1
# Description:
#   Bug 1971161 [Wayland] Refresh monitor setup on wp_color_management_output_v1::image_description_changed signal r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D256433
# ==============================================================================

diff -r e977efc8bc30 -r f562693b5b30 widget/gtk/ScreenHelperGTK.h
--- a/widget/gtk/ScreenHelperGTK.h	Tue Jul 08 11:41:56 2025 +0000
+++ b/widget/gtk/ScreenHelperGTK.h	Tue Jul 08 11:41:56 2025 +0000
@@ -8,13 +8,14 @@
 #define mozilla_widget_gtk_ScreenHelperGTK_h
 
 #include "mozilla/widget/ScreenManager.h"
-
 #include "gdk/gdk.h"
+#ifdef MOZ_WAYLAND
+#  include "nsWaylandDisplay.h"
+#endif
 
 class nsWindow;
 
 namespace mozilla::widget {
-
 class ScreenGetterGtk;
 
 class ScreenHelperGTK final : public ScreenManager::Helper {